### PR TITLE
some small initial changes and adds

### PR DESCRIPTION
**Pull request by gindij**
added index.html, lectures.html, changed a few things on the default.html layout

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -86,8 +94,8 @@
 
       <footer class="text-center text-muted">
         <hr/>
-        Last updated {{ site.time | date: "%B %d, %Y" }}. <br/>
-	The source code is on <a href="https://github.com/callison-burch/data-structures-and-algorithms">github <i class="fa fa-github-alt"></i></a>.
+        <script type="text/javascript">document.write("Last updated " + document.lastModified + ".");</script> <br/>
```
Comment (jabbany) [inline _layouts/default.html]: I think it makes more sense to let Jekyll staticly generate the last update time. We could rename this to be something like "Last generated"

@lyze Thoughts?

**Comment by gindij** [comment]
sorry about the confusion, used jekyll this time

File: _layouts/default.html [hunk 2]
```diff
@@ -46,22 +47,22 @@
...
+            <li id="tutoring"><a href="http://www.vpul.upenn.edu/tutoring/">Tutoring</a></li>
             <li id="syllabus"><a href="syllabus.html">Syllabus</a></li>
-            <li id="syllabus"><a href="resources.html">Resources</a></li>
+            <li id="resources"><a href="resources.html">Resources</a></li>
             <li id="homework">
               <a class="dropdown-toggle" data-toggle="dropdown" href="#">Homework Assignments <span class="caret"></span></a>
               <ol class="dropdown-menu">
                 <li><a href="hw1.html">HW1 (due XXX)</a></li>
-	      </ol>
+        </ol>
```
Comment (jabbany) [inline _layouts/default.html]: Indent.

File: _layouts/default.html [hunk 3]
```diff
@@ -46,22 +47,22 @@
...
             </li>
             <li id="labs">
               <a class="dropdown-toggle" data-toggle="dropdown" href="#">Labs <span class="caret"></span></a>
               <ol class="dropdown-menu">
                 <li><a href="lab1.html">Lab 1 (due XXX)</a></li>
-	      </ol>
+        </ol>
```
Comment (jabbany) [inline _layouts/default.html:69]: Indent

File: _layouts/default.html [hunk 4]
```diff
@@ -87,7 +88,7 @@
       <footer class="text-center text-muted">
         <hr/>
         Last updated {{ site.time | date: "%B %d, %Y" }}. <br/>
-	The source code is on <a href="https://github.com/callison-burch/data-structures-and-algorithms">github <i class="fa fa-github-alt"></i></a>.
+  The source code is on <a href="https://github.com/callison-burch/data-structures-and-algorithms">github <i class="fa fa-github-alt"></i></a>.
```
Comment (jabbany) [inline _layouts/default.html:95]: Indent

File: index.md [hunk 5]
```diff
@@ -8,29 +8,29 @@ active_tab: main_page
 ---
 
 
-Course number
+Course number:
```
Comment (jabbany) [inline index.md:11]: Doesn't this cause two `:`s to be added? There is one at the beginning of the next line.

**Comment by jabbany** [comment]
Also, squash these two commits into one so we won't need to incorporate the addition and deletion of the two html files into the history.

http://stackoverflow.com/questions/6102297/how-do-i-clean-up-my-github-fork-so-i-can-make-clean-pull-requests

File: index.md (hunk 5 above)
Comment (gindij) [inline index.md:11]: No it doesn't.

Comment (jabbany) [inline index.md:11]: I see. It's fine to keep these.

Comment (gindij) [inline index.md:11]: I think I squashed the commits.

File: resources.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+img_link: http://algs4.cs.princeton.edu/home/
+caption: The course textbook
+title: resources
+active_tab: resources
+---
+
+
+<div>
+    {% for resource in site.data.resources %}
+    <h3>{{ resource.header }}</h3>
```
Comment (lyze) [inline resources.md]: Indent.

File: resources.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        <ul>
+            {% for link in resource.links %}
+            <li><a href="{{ link.link }}">{{ link.text }}</a></li>
```
Comment (lyze) [inline resources.md]: Indent.

**Comment by lyze** [comment]
While you're at it, can you also clean up the trailing whitespace too? Fix those nits, and LGTM.

**Comment by jabbany** [comment]
Great job with the markdown. I'll let David have one last check before we merge this in 
:)

File: resources.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+layout: default
+img: textbook.jpg
+img_link: http://algs4.cs.princeton.edu/home/
+caption: The course textbook
+title: resources
+active_tab: resources
+---
+
+<div>
+	<ul class="resource-header-list">
```
Comment (lyze) [inline resources.md:11]: Looks like the indentation got messed up here.

File: labs.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+<table class="table table-striped"> 
+  <tbody>
+    <tr>
+      <th>Name</th>
+      <th>Materials</th>
+      <th>Due Date</th>
+    </tr>
+    {% for lab in site.data.labs %}
+    <tr style="text-align: left">
```
Comment (lyze) [inline labs.md]: Indent the `{% for %}` and also the body of it too.

File: homework.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+<table class="table table-striped"> 
+  <tbody>
+    <tr>
+      <th>Name</th>
+      <th>Materials</th>
+      <th>Due Date</th>
+    </tr>
+    {% for homework in site.data.homeworks %}
```
Comment (lyze) [inline homework.md]: Indent the {% for %} and also the body of it too.

File: resources.md (hunk 8 above)
Comment (gindij) [inline resources.md:11]: I'm not sure how to fix the indentation for this one. In sublime (which I'm
using to edit), the indentation is fine.

Comment (jabbany) [inline resources.md:11]: @gindij 
It seems you're mixing tabs and spaces which makes for really bad indentation when viewed on Github or in a console editor etc. I don't use Sublime, but I think most sane editors allow you to set it to use one of tabs or spaces for it's auto-indentation. 

I have no personal preference over either (some people do), just don't mix the two. Go around all the files and fix any mixed tabs/spaces issues (easy way is to copy a tab or space character and then search for it in the file, the editor usually highlights the whitespace so you can fix any problems). There might also be plugins/tools that do this for you.

File: _layouts/default.html [hunk 11]
```diff
@@ -10,6 +10,7 @@
     <link href="./dist/css/bootstrap.min.css" rel="stylesheet">
     <link href="assets/css/class.css" rel="stylesheet">   
     <link href="assets/css/syntax.css" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/style.css" rel="stylesheet">
```
Comment (jabbany) [inline _layouts/default.html:13]: Remove duplicated `rel=` attribute

**Pull request merged**